benches/hvm_cpu.rs

```diff
@@ -103,8 +127,8 @@ fn interact_benchmark(c: &mut Criterion) {
     group.bench_function(name, |b| {
       b.iter_batched(
         || net.clone(),
-        |mut net| black_box(net.interact(&book, rdx_a, rdx_b)),
-        criterion::BatchSize::SmallInput
+        |net| black_box(net).interact(black_box(&book), black_box(rdx_a), black_box(rdx_b)),
```

Comment:
hmm, I think the `black_box` should wrap everything. see criterion doc

Reply:
![image](https://github.com/HigherOrderCO/hvm-core/assets/19491161/ef610234-10bb-445e-bdeb-237034232442)
The way I read about how black_box should be used, is that the inputs of the function should be boxed. This function even returns `()` and was triggering a clippy_lint about passing unit to a function. Am I interpreting this wrong?

Reply:
Or should both the inputs and unit output be boxed?

Reply:
I see, I was wrapping in the other functions too and not on this one. I re-added the black_box here